app/main.py

```diff
@@ -1 +1,110 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    # Check if input command format is correct
+    if not is_input_data_correct(command):
+        return
+
+    # Get correct paths for source and destination files
+    origin_input_path, dest_input_path = command.split()[1:]
+    dest_short_path, dest_file_name = os.path.split(dest_input_path)
+    origin_short_path, origin_file_name = os.path.split(origin_input_path)
+    if not dest_file_name:
+        dest_full_path = os.path.join(dest_short_path, origin_file_name)
+    else:
+        dest_full_path = dest_input_path
+
+    # If new location is not provided:
+    # Rename source file
+    if not origin_short_path and not dest_short_path:
+        rename_file(origin_file_name, dest_file_name)
+        return
+
+    # If new location is provided:
+    # Open source file for reading
+    try:
+        origin_file = open(origin_input_path, "r")
+    except FileNotFoundError:
+        print(f"No source file with such name: {origin_input_path}")
+        return
+    except PermissionError:
+        print(f'"{origin_file_name}" is a folder')
+        return
+
+    # Create folder tree given in the destination path
+    try:
+        create_folder_tree(dest_short_path)
```

Comment:
```suggestion
        if dest_short_path:
              os.makedirs(dest_short_path, exist_ok=True)
```
Simplify code please

Reply:
thanks!